### DISCUSSION TITLE
New block for "my page" of "tree home page"

### DISCUSSION THREAD

**Discussion by hartenthaler** [vesta-webtrees-2-custom-modules/vesta_extended_relationships#95] [Q&A]
There is a new block for use on my-page and tree-home-page available: "Vesta Extended Relationships". It shows only the logged-in users. How can I show an extended relationship diagram? What is the difference to the existing block "Who is online"?

**Comment by ric2016** [comment]
> What is the difference to the existing block "Who is online"?

There is an additional link "Relationship to me" shown for each logged-in user ([see here](https://github.com/vesta-webtrees-2-custom-modules/vesta_extended_relationships/issues/90)), if possible.

> How can I show an extended relationship diagram?

By configuring the chart (as before, nothing has changed here).
